iris/src/iris/tracing/langfuse_tracer.py

```
@@ -0,0 +1,466 @@
+                logger.debug(
+                    "LangFuse observe failed, executing without tracing: %s", e
+                )
+                return func(*args, **kwargs)
+
+        return wrapper  # type: ignore
+
+    return decorator
+
+
+def get_langchain_callback(
+    ctx: Optional[TracingContext] = None,
```

Ruff (0.14.10)

Unused function argument: `ctx`

(ARG001)